library/src/scala/annotation/experimental.scala

```diff
@@ -0,0 +1,4 @@
+package scala.annotation
+
+/** An annotation that can be used to mark a definition as experimental. */
+class experimental extends StaticAnnotation
```

Comment:
Until we figure out if this should be exposed to library authors and what it should do for them, we might want to mark experimental itself as experimental (bu this might lead to a cycle in the typechecker, so it'd be better to hardcode that specific check instead)

Reply:
Done